build/ci/check_demo_pr_checks.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -xe
+set +o pipefail
+
+export passed_k8s_checks=true
+export passed_openshift_checks=true
+
+wait_for_checks_to_complete(){
+  community_operators_branch=$1
+  all_checks_passed=true
+  export repo_pr=`gh pr list --repo $github_csiblock_community_operators_repository | grep $community_operators_branch`
+  if [[ "$repo_pr" == *"$community_operators_branch"* ]]; then
+    sleep 5
+    while [ "$(gh pr checks $community_operators_branch --repo $github_csiblock_community_operators_repository | grep -i pending | wc -l)" -gt 0 ]; do
+      sleep 15
+    done
+    summary_counter=0
+    export search_summary_test=`gh pr checks $community_operators_branch --repo $github_csiblock_community_operators_repository | grep -i summary`
+    while [ "$search_summary_test" != *"pending"* ] && [ $summary_counter -lt 20 ]; do
+      sleep 1
+      export search_summary_test=`gh pr checks $community_operators_branch --repo $github_csiblock_community_operators_repository | grep -i summary`
+      ((summary_counter=summary_counter+1))
+      if [[ "$search_summary_test" == *"pass"* ]]; then
+        summary_counter=20
```

Comment:
what does `20` mean here?

Reply:
to break the while loop and "tell" the `if` at line 26 that the summary test has passed

Reply:
I don't see how this is related to the number `20`.
sounds like a `is_summary_test_passed` boolean to me

Reply:
done